### DISCUSSION TITLE
RPIZeroW - Spotify don't turn ON

### DISCUSSION THREAD

**Discussion by czosnekltd** [rern/rAudio#237] [Bug Reports]
In the Feauters menu, after enabling the Spotify Checkbox option, it turns on for a while, but after a while returns to the OFF position. After connecting through SSH, I have the following messages 

rAudio:~  #  pacman -Q spotifyd; systemctl status spotifyd
spotifyd 0.3.0-1
Unit spotifyd.service could not be found.
rAudio:~  #  pacman -Q spotifyd; systemctl enable spotifyd
spotifyd 0.3.0-1
Failed to enable unit: Unit file spotifyd.service does not exist.
rAudio:~  #

*********************************
rAudio 1 • 20210302
5.10.17-2-ARCH armv6l
Raspberry Pi Zero W Rev 1.1
BCM2835 • 455MiB
ARM1176 @ 1GHz

**Comment by rern** [comment]
Try:
`ln -s /usr/lib/systemd/{user,system}/spotifyd.service`
Then enable Spotify again.
